lib/SPIRV/libSPIRV/spirv_internal.hpp

```diff
@@ -113,7 +115,8 @@ enum InternalCapability {
   ICapabilityJointMatrixPackedInt2ComponentTypeINTEL = 6438,
   ICapabilityJointMatrixPackedInt4ComponentTypeINTEL = 6439,
   ICapabilityCacheControlsINTEL = 6441,
-  ICapRegisterLimitsINTEL = 6460
+  ICapRegisterLimitsINTEL = 6460,
```

Comment:
Just curious. Why do we not have this Capability in the Khronos main branch? Thanks

Reply:
It can actually be updated, I see https://github.com/KhronosGroup/SPIRV-Registry/pull/235/files is published